src/main/twirl/uk/gov/hmrc/play/views/layouts/header_nav.scala.html

```diff
@@ -18,29 +18,31 @@
   showBetaLink: Boolean,
   navLinks: Option[Html])
 
-<div class="header-proposition">
-    <div class="content">
-        @if(navLinks) {
-        <a href="#proposition-links" class="js-header-toggle menu">Menu</a>
-        }
-        <nav id="proposition-menu" class="header__menu" role="navigation">
-
-            @if(navTitleLink){
-                <a href="@navTitleLink" class="header__menu__proposition-name">@navTitle</a>
-            }else{
-                <span class="header__menu__proposition-name">@navTitle</span>
+@if(navTitle || navLinks || showBetaLink) {
```

Comment:
Please use the following methods for the two options inside if the if conditional:
- `navTitle.isDefined`
- `navLinks.isDefined`